format/src/main/scala/org/scalafmt/internal/SyntaxTokens.scala

```diff
@@ -269,8 +279,33 @@ object SyntaxTokens {
   }
 
   // == Defn ==
+  implicit class XtensionCtorPrimarySyntax(val tree: Ctor.Primary)
+      extends AnyVal {
+    def tokensCommaCtor: List[List[Comma]] =
+      tree.paramss.map(commaSeparated0(tree))
+
+    def tokensImplicit: Option[KwImplicit] = {
```

Comment:
Is this method necessary? There's no `tokensVar` or `tokenVal` to handle `class Foo(var x: Int, val y: String)`. In addition, this should be implementable with `Mod.Implicit().tokens` and if that doesn't work then it's a bug in scalameta.

Reply:
I propose to remove this method.